data-sets-api-server/src/test/java/org/zowe/unix/files/services/zosmf/ListUnixDirectoryZosmfRunnerTest.java

```diff
@@ -31,10 +31,22 @@
     
     @Test
     public void get_unix_directory_list_should_call_zosmf_and_parse_response_correctly() throws Exception {
-        UnixDirectoryChild file = UnixDirectoryChild.builder().name("FileA").type(UnixEntityType.FILE)
-                .link("http://localhost/a/directory/FileA").build();
-        UnixDirectoryChild directory = UnixDirectoryChild.builder().name("DirectoryA").type(UnixEntityType.DIRECTORY)
-                .link("http://localhost/a/directory/DirectoryA").build();
+        UnixDirectoryChild file = UnixDirectoryChild
+            .builder()
+            .name("FileA")
+            .type(UnixEntityType.FILE)
+            .size(12345)
+            .lastModified("2019-02-13T16:04:19")
+            .link("http://localhost/a/directory/FileA")
+            .build();
+        UnixDirectoryChild directory = UnixDirectoryChild
+            .builder()
+            .name("DirectoryA")
+            .type(UnixEntityType.DIRECTORY)
```

Comment:
Update the size and last modified to be different for the the different files please